upgrade_test.py

```diff
@@ -1244,3 +1247,167 @@ def _update_argus_upgraded_version(self, node, new_version):
                 self.log.warning("Couldn't extract version from %s", new_version)
         except Exception as exc:  # pylint: disable=broad-except
             self.log.exception("Failed to save upgraded Scylla version in Argus", exc_info=exc)
+
+
+class UpgradeCustomTest(UpgradeTest):
+    def test_custom_profile_rolling_upgrade(self):
+        """
+        Run a load of a custom profile.
+        Upgrade half of nodes in the cluster, and run rollback.
+        Upgrade all nodes to new version in the end.
+        """
+        cs_user_profiles = self.prepare_data_before_upgrade()
+        self._custom_profile_rolling_upgrade(cs_user_profiles=cs_user_profiles)
+
+    def test_custom_profile_sequential_rolling_upgrade(self):
+        """
+        Run a load of a custom profile.
+        Upgrade half of nodes in the cluster, and run rollback.
+        Upgrade all nodes to new version in the end.
+        """
+        cs_user_profiles = self.prepare_data_before_upgrade()
+        # Temporary solution. If we want to have sequential upgrade test, we will need to change the test parameters and build upgrade path
+        new_version = "2022.1.23"
+        new_scylla_repo = \
+            "http://downloads.scylladb.com/unstable/scylla-enterprise/enterprise-2022.1/deb/unified/2023-09-10T15:52:16Z/scylladb-2022.1/" \
```

Comment:
why not using the official links ? why picking it from unstable ?

Reply:
https://s3.amazonaws.com/downloads.scylladb.com/deb/ubuntu/scylla-2022.1.list

Reply:
I took it from official rolling upgrade test

Reply:
changed

Reply:
in the jobs, its a temporary current repo with only one version, i.e.not the official repos